packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2718,7 +2718,8 @@ class FractionallySizedBox extends SingleChildRenderObjectWidget {
   /// unmodified.
   final double? heightFactor;
 
-  /// How to align the child.
+  /// How to align the child. If [Alignment] property is specified, then [FractionallySizedBox]
```

Comment:
The convention for API docs is to start with a single sentence that's an overview of the property/method, then a blank line, then whatever else is needed.

That said, I don't think this line is particularly helpful. I think what you're trying to say is that, for the alignment parameter to have an effect, the child's parent must be larger than the child. That will happen if the constraints inherited by the parent force it to be so.

A simple interactive example would be helpful.

Reply:
Great thanks!